### ISSUE TITLE
How to Choose Image from Galllery in iOS 7...?? 

### ISSUE THREAD

**Issue by sohaib770** [kif-framework/KIF#466]
I am using the following function to select and image from album
[tester choosePhotoInAlbum:@"Recently Added" atRow:0 column:0];
But error occurred  "Failed to find accessibility element with the label "Choose Photo".
